### DISCUSSION TITLE
Inventory/Worn Items customizer sidebar panel

### DISCUSSION THREAD

**Discussion by carleyenterprises** [runelite/runelite#16554] [Ideas]
I have an idea for an Inventory and Worn Items sidebar panel that allows users to create virtual gear setups and corresponding inventories. 

The sidebar would include customizable inventory and worn item templates that could be populated with item sprites for any gear or items selected by the user. The custom templates would not impact the user's in-game inventory or worn items.

**Comment by Nightfirecat** [comment]
There's a hub plugin which does this already, [Inventory Setups](https://runelite.net/plugin-hub/show/inventory-setups)
